pkg/network/usm/ebpf_ssl_test.go

```diff
@@ -73,17 +74,46 @@ func TestNativeShortLivedProcess(t *testing.T) {
 	libmmap := filepath.Join(curDir, "testdata", "site-packages", "ddtrace")
 	lib := filepath.Join(libmmap, fmt.Sprintf("libssl.so.%s", runtime.GOARCH))
 
+	var pidMap sync.Map
+	utils.RegisterHook = func(pid uint32) {
+		pidMap.Store(pid, struct{}{})
+	}
+	t.Cleanup(func() {
+		// Will be called after monitor is stopped (due to t.Cleanup() ordering)
+		// so should be race-free.
+		utils.RegisterHook = nil
+	})
+
 	monitor := setupUSMTLSMonitor(t, cfg)
 	require.NotNil(t, monitor)
 
+	pids := make([]uint32, 10)
```

Comment:
```suggestion
	pids := make([]uint32, 0, 10)
```
otherwise you'll end up with 20 entries in `pids` where the first 10 entries are `0`